liblangutil/Token.h

```diff
@@ -314,6 +314,13 @@ namespace TokenTraits
 	constexpr bool isTimeSubdenomination(Token op) { return op == Token::SubSecond || op == Token::SubMinute || op == Token::SubHour || op == Token::SubDay || op == Token::SubWeek || op == Token::SubYear; }
 	constexpr bool isReservedKeyword(Token op) { return (Token::After <= op && op <= Token::Unchecked); }
 
+	constexpr bool isYulToken(Token tok)
+	{
+		return tok == Token::Function || tok == Token::Let || tok == Token::If || tok == Token::Switch || tok == Token::Case
+		|| tok == Token::Default || tok == Token::For || tok == Token::Break || tok == Token::Continue /* || tok == Token::Leave */
+		|| tok == Token::TrueLiteral || tok == Token::FalseLiteral;
```

Comment:
```suggestion
			|| tok == Token::TrueLiteral || tok == Token::FalseLiteral;
```

Reply:
Actually the style of this return is wrong. `||` should be at the end of the previous line, and the two lines after the return should have an extra tab

Reply:
Fixed.